src/components/BookList/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from 'react';
+
+import { getBooks } from '../../services/BookServices';
+import defaultImageBook from '../../assets/book.jpg';
+
+const BookList = () => {
+  const [loading, setLoading] = useState(false);
+  const [withoutBooks, setWithoutBooks] = useState(false);
+  const [bookList, setBookList] = useState([]);
+
+  useEffect(() => {
+    setLoading(true);
+    getBooks()
+      .then(
+        response => {
+          if (response && response.data && response.data.length) {
+            const list = response.data.map(book => (
+              <div className="column m-left-2 m-bottom-2" key={book.id}>
+                <img src={book.image.url || defaultImageBook} alt="" />
+                <span>{book.title}</span>
+                <span>{book.author}</span>
+              </div>
+            ));
```

Comment:
Keep the html in the render. Here just assign the value to the list and map it in the return below